### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by divadwong68** [SinTan1729/chhoto-url#29] [Ideas]
With all apps now getting dark mode options like it's a mandatory feature, it would be nice if chhoto gets this feature.

**Comment by SinTan1729** [comment]
I don't have any plans to do it, since the website looks nice with Dark Reader. But I'm open to accepting PRs if someone decides to add the feature.

**Reply by TC97AU** [reply]
It might be worth noting in the readme that those screenshots are using Dark Reader, just so people aren't surprised with a flashbang!

**Reply by SinTan1729** [reply]
@TC97AU Good idea.

**Reply by SinTan1729** [reply]
I decided to add dark automatic mode support in babf3d891180c429a35b83d549fe6e1ad976fddb. Let me know if you find any issues.
